docs/realtime/changes/index.de.md

```diff
@@ -0,0 +1,104 @@
+
+!!! note ""
+
+    Die offizielle Spezifikation, Referenz und Dokumentation sind in englischer Sprache verfasst. Weicht eine Translation in eine andere Sprache vom englischen Original ab, hat letztere Vorrang. Die gesamte Kommunikation erfolgt in englischer Sprache.
+
+<!-- <br><div class="landing-page">
+    <a class="button" href="../process">Prozess zur Änderung der Spezifikation</a><a class="button" href="../guiding-principles">Leitprinzipien</a><a class="button" href="../revision-history">Historie der Revision</a><a class="button" href="../extensions">Echtzeit-Erweiterungen</a>
+</div> -->
+
+## Aktive Proposals![](../../assets/pr-active.svg)
+
+<!-- Aktive Vorschläge für neue Funktionen in GTFS Realtime.  -->
+
+Beteiligen Sie sich an den Diskussionen auf [Github](https://github.com/google/transit/pulls)!
+
+<!-- <div class="row">
+    <div class="active-container">
+        <h3 class="title"><a class="no-icon" href="https://github.com/google/transit/pull/332" target="_blank">Hinzufügen von cause_detail und effect_detail zu Alerts</a></h3>
+        <p class="maintainer">#332 eröffnet am 31. Mai 2022 von <a class="no-icon" href="https://github.com/mckenzie-maidl-ibigroup" target="_blank">mckenzie-maidl-ibigroup</a></p>
+    </div>
+</div>
+<div class="row"></div> -->
+
+<!-- <div class="row no-active">
+    <div class="no-active-container">
+        <h3 class="title">Derzeit gibt es keine aktiven Vorschläge für GTFS Realtime.</h3>
+        <p class="prompt">Haben Sie einen Vorschlag? &ensp;➜&ensp; Öffnen Sie eine <a href="https://github.com/google/transit/pulls" target="_blank">Pull-Anfrage</a>.</p>
+    </div>
+</div>
+<div class="row"></div> -->
+
+## Kürzlich zusammengefasste Vorschläge![](../../assets/pr-merged.svg)
+
+Kürzlich zusammengeführte Vorschläge, die nun Bestandteil der [offiziellen GTFS Realtime-Spezifikation](../reference) sind. Weitere Informationen finden Sie in der vollständigen [Revisionshistorie](../process#revision-history).
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/332" class="no-icon" target="_blank">Hinzufügen von cause_detail und effect_detail zu Alerts</a></h3>
+        <p class="maintainer">#332 von <a href="https://github.com/mckenzie-maidl-ibigroup" class="no-icon" target="_blank">mckenzie-maidl-ibigroup</a> wurde zusammengeführt am 26. Juni 2022</p>
+    </div>
+    <div class="featurelist">
+        <ul>
+            <li>Fügt Beschreibungen zu Ursache und Wirkung von Alerts hinzu</li>
+        </ul>
+    </div>
+</div>
+
+<div class="row">
+    <div class="leftcontainer">
+        <h3 class="title"><a href="https://github.com/google/transit/pull/340" class="no-icon" target="_blank">GTFS-rt : Aktualisierung des Rollstuhlzugangs</a></h3>
+        <p class="maintainer">#340 von <a href="https://github.com/flaktack" class="no-icon" target="_blank">flaktack</a> wurde am 25. Juli 2022 zusammengeführt.</p>
```

Comment:
As above, the wording could be improved. Right now, one can understand it when thinking about it, but it doesn't follow the "natural flow" as one might express this in German.

Reply:
Thank you for this suggestion! Do you have any recommendation for how the structure could be improved?